api/proto/teleport/accessmonitoringrules/v1/access_monitoring_rules.proto

```diff
@@ -45,6 +45,8 @@ message AccessMonitoringRuleSpec {
   string condition = 3;
   // notification defines the plugin configuration for notifications if rule is triggered.
   Notification notification = 4;
+  // automatic_approval defines the plugin configuration for automatic approvals.
+  AutomaticApproval automatic_approval = 5;
```

Comment:
Can you document here if both can be set, if it's like a one-off? What happens when we set both, can we set them to different plugins?

Reply:
I've updated the docs a bit. Both values may be set, and they can be set to different plugins. If both values are set, notifications and automatic approvals should be handled for the same set of access events.